cmd/dep/godep_importer_test.go

```diff
@@ -395,3 +338,13 @@ func equalImports(a, b []godepPackage) bool {
 
 	return true
 }
+
+// projectRootPtr takes a ProjectRoot value and returns a pointer.
+func projectRootPtr(p gps.ProjectRoot) *gps.ProjectRoot {
```

Comment:
I've laid out in the comments an alternative way to simplify the test cases and remove these helper functions. Let me know if they make sense.

I don't mean to hate on them, but if we can reorganize the tests to not need them, that would make me happy. 😀